src/QMCWaveFunctions/OrbitalBase.h

```diff
@@ -606,7 +606,7 @@ struct OrbitalBase: public QMCTraits
     app_error() << "Required CUDA functionality not implemented. Contact developers.\n";
     abort();
   }
-#endif
+#endif //defined(QMC_CUDA) && !defined(ENABLE_SOA)
```

Comment:
Same issue (#endif comment does not match #if statement).

Reply:
see above